openhtf/io/output/__init__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2014 Google Inc. All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Openhtf output modules."""
+
+import json_factory
+import mfg_inspector
```

Comment:
Little functional difference here, but Google's style guide prefers us to use absolute imports for these (`openhtf.io.output.json_factory`) rather than using relative imports.

> Do not use relative names in imports. Even if the module is in the same package, use the full package name. This helps prevent unintentionally importing a package twice.

https://google.github.io/styleguide/pyguide.html#Imports